libexec/ld.elf_so/arch/hppa/hppa_reloc.c

```diff
@@ -625,6 +625,10 @@ _rtld_relocate_plt_lazy(Obj_Entry *obj)
 		Elf_Addr *where = (Elf_Addr *)(obj->relocbase + rela->r_offset);
 		Elf_Addr func_pc, func_sl;
 
+		/* skip R_PARISC_NONE entries */
+		if (ELF_R_TYPE(rela->r_info) == R_TYPE(NONE))
+			continue;
+
```

Comment:
question: I see you're adding more stuff. Is this a merge from newer revisions or are you, in fact, fixing stuff?

Reply:
I didn't add this stuff, not sure why this PR is getting these other commits when it says that the branch is just 3 commits ahead. I'll try to fix it and if not possible, I open a new PR.